payjoin/src/send/mod.rs

```diff
@@ -465,15 +466,123 @@ pub(crate) mod test {
             }),
             min_fee_rate: FeeRate::ZERO,
             payee,
-        }
+        })
     }
 
     #[test]
-    fn official_vectors() {
-        let original_psbt = Psbt::from_str(ORIGINAL_PSBT).unwrap();
+    fn test_determine_fees() -> Result<(), BoxError> {
+        let original_psbt = Psbt::from_str(ORIGINAL_PSBT)?;
```

Comment:
Shouldn't we use `unwrap()` here? Since these are testcases, I think it's better to panic and unwind the stack, rather than returning a `Err`, more verbose in case it actually fails.